vortex/src/main/java/org/apache/iceberg/vortex/VortexIterable.java

```diff
@@ -169,7 +169,7 @@ private static Map<String, String> s3PropertiesFromHadoopConf(Configuration hado
     return properties.asProperties();
   }
 
-  static final String ACCESS_KEY_PREFIX = "fs.azure.account.key.";
+  static final String ACCESS_KEY_PREFIX = "fs.azure.account.key";
```

Comment:
We can actually set `fs.azure.account.key` and it seems to be used for all storage accounts, helpful for tests or when you know all of the job's inputs only come from one storage account